crates/bevy_reflect/src/list.rs

```diff
@@ -15,6 +15,9 @@ pub trait List: Reflect + Array {
     /// Appends an element to the list.
     fn push(&mut self, value: Box<dyn Reflect>);
 
+    /// Removes the last element from a list and returns it, or [`None`] if it is empty.
```

Comment:
Nit: More consistent articles with the `push` method
```suggestion
    /// Removes the last element from the list and returns it, or [`None`] if it is empty.
```

---

Do you think these methods should note that the actual implementation may differ for different data structures? Since, for example a queue-like structure would pop the _first_ element rather than the last (depending on how you look at it). Maybe a super basic note that just tells the user to be aware of this in a general sense?